lib/components/CheckboxButtons/CheckboxButtons.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { expect, fn, getAllByRole, within, userEvent } from '@storybook/test';
+import React, { useState } from 'react';
+
+import { CheckboxButtons } from './CheckboxButtons';
+
+const itemData = [
+	['Tyre puncture repair', '+28.00'],
+	['Roadworthy certificate', '+89.85'],
+	['Re-gas air-conditioning', '+99.00'],
+	['Windscreen wipers - pair', '+45.92'],
+];
+
+const meta: Meta<typeof CheckboxButtons> = {
+	title: 'Components/Checkbox Buttons',
```

Comment:
Should this be nested inside the `Inputs` folder?

Reply:
I'd like to flatten the sidebar and maybe use the new storybook tags to filter by use case